### DISCUSSION TITLE
Get biogenic carbon from eGRID instead of GHGRP

### DISCUSSION THREAD

**Discussion by Greatest125** [USEPA/standardizedinventories#116] [Ideas]
Currently, biogenic CO2 comes from GHGRP and fossil CO2 comes from eGRID. Since more facilities are in eGRID than GHGRP, this means that biogenic CO2 for facilities that aren't in GHGRP is being left out, and therefore CO2 emissions are being undercounted. Also, data from eGRID has a reliability score of 2 while data from GHGRP has a reliability score of 5.

Therefore, I think StEWI should get biogenic CO2 data from eGRID instead (since more facilities will get biogenic CO2 data plus it'll be more reliable). This is possible by subtracting the UNCO2 (unadjusted CO2 -- so total CO2 emissions) by PLCO2AN (adjusted CO2 -- so only fossil CO2). What do you think?

Thank you!

**Comment by WesIngwersen** [comment]
@Greatest125 this is some good insight. Could you shoot me an [email ](https://github.com/WesIngwersen) and, you, @bl-young and I can setup a time to discuss live?

**Comment by bl-young** [comment]
The selection of specific inventories when combined is dictated by this dictionary:
https://github.com/USEPA/standardizedinventories/blob/698a0b316f5cd999f33ae8ddffe0a2fca94f2bf9/stewicombo/globals.py#L28-L32

The stewi data from GHGRP does distinguish the biogenic CO2, but as you say when combined with eGRID that data is dropped in place of the data from eGRID, which does not track biogenic CO2.

In order to get the biogenic CO2 from eGRID I believe we would want to use the "BIOCO2" field ("Plant annual CO2 biomass emissions (tons)"). We currently use only "PLCO2AN" (annual adjusted CO2 emissions). In many cases, BIOCO2 + PLCO2AN = UNCO2, however there are other reasons why the emissions differ from the unadjusted value besides bio-based emissions. So calculating that data as UNCO2 - PLCO2AN would be an overestimate.

I'll note that the biogenic CO2 data is not carried forward into the other tabs so we would not be able to validate that datapoint like we do in the case of the other flows.

**Comment by bl-young** [comment]
@Greatest125 I have [created an issue](https://github.com/USEPA/standardizedinventories/issues/117) for this enhancement. We welcome contributions from users (fork from develop and submit a pull request back to develop). Otherwise I will get to this when I am able.

**Reply by Greatest125** [reply]
Sounds good. Note that the "BIOCO2" field was added in the 2018 release of eGRID. Prior to 2018, should we use UNCO2 - PLCO2AN (I was told ~2010 by someone in EPA that was the best way to calculate biogenic) or data from GHGRP?

Thanks

**Reply by bl-young** [reply]
Oh ok good to know I didn't look back at older years. At first glance that seems reasonable.
